sdk/keyvault/azure-security-keyvault-jca/src/main/java/com/azure/security/keyvault/jca/implementation/JREKeyStoreFactory.java

```diff
@@ -85,7 +84,7 @@ private static Path getConfiguredKeyStorePath() {
 
     @SuppressWarnings("removal")
     private static String  privilegedGetProperty(String theProp, String defaultVal) {
-        return AccessController.doPrivileged(
+        return java.security.AccessController.doPrivileged(
```

Comment:
Changes to this file can be reverted.

Reply:
So, this is a compiler warning fix for Java 17. Apparently when you suppress warnings for `removal`, or anything, on the class it doesn't cover import statements and import statements can't be annotated with `SuppressWarnings`. Making the usage of the class fully-qualified allows for the method to suppress the warning and fix the compilation warning.